timemachine/fe/bar.py

```diff
@@ -322,7 +322,7 @@ def pair_overlap_from_ukln(u_kln: NDArray) -> float:
 
     """
     u_kn, N_k = ukln_to_ukn(u_kln)
-    return pymbar.mbar.MBAR(u_kn, N_k).compute_overlap()["scalar"]  # type: ignore
+    return max(0, 2 * pymbar.MBAR(u_kn, N_k).compute_overlap()["matrix"][0, 1])  # type: ignore
```

Comment:
Noting that this returns 0 when `compute_overlap()` returns NaN. Might want to note this in the docstring or use `np.maximum` instead?